blazor/autocomplete/filtering.md

```diff
@@ -37,6 +37,19 @@ The below code demonstrates the filtering functionality with [ODataAdaptor](http
 
 ![Blazor AutoComplete with Remote Data filtering](./images/filtering/blazor_autocomplete_remote-data.png)
 
+## DebounceDelay in Filtering
+
+You can use the `DebounceDelay` property for filtering, enabling you to set a delay in milliseconds. This functionality helps reduce the frequency of filtering as you type, enhancing performance and responsiveness for a smoother user experience.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/debounce-delay.razor %}
+
+{% endhighlight %}
+
+![Blazor AutoComplete with DebounceDelay in filtering](./images/filtering/blazor_autocomplete_debounce-delay.gif)
```

Comment:
Changes the gif, it should show case debouncing

Reply:
changed sir